source/crud/tests/unified/aggregate-out-readConcern.yml

```diff
@@ -0,0 +1,173 @@
+# This file was created automatically using mongodb-spec-converter.
+# Please review the generated file, then remove this notice.
+
+description: aggregate-out-readConcern
+schemaVersion: '1.1'
+createEntities:
+  -
+    client:
+      id: &client0 client0
+      useMultipleMongoses: false
+      uriOptions:
+        retryReads: false
+      observeEvents:
+        - commandStartedEvent
+  -
+    database:
+      id: &database0 database0
+      client: client0
+      databaseName: &database_name crud-v2
+  -
+    collection:
+      id: &collection0 collection0
+      database: database0
+      collectionName: &collection_name test_aggregate_out_readconcern
+  -
+    collection:
+      id: &collection_readConcern_majority collection_readConcern_majority
+      database: database0
+      collectionName: *collection_name
+      collectionOptions:
+        readConcern: { level: "majority" }
+  -
+    collection:
+      id: &collection_readConcern_local collection_readConcern_local
+      database: database0
+      collectionName: *collection_name
+      collectionOptions:
+        readConcern: { level: "local" }
+  -
+    collection:
+      id: &collection_readConcern_available collection_readConcern_available
+      database: database0
+      collectionName: *collection_name
+      collectionOptions:
+        readConcern: { level: "available" }
+  -
+    collection:
+      id: &collection_readConcern_linearizable collection_readConcern_linearizable
+      database: database0
+      collectionName: *collection_name
+      collectionOptions:
+        readConcern: { level: "linearizable" }
+runOnRequirements:
+  -
+    minServerVersion: 4.1.0
+    topologies:
+      - replicaset
+      - sharded
+initialData:
+  -
+    collectionName: *collection_name
+    databaseName: *database_name
+    documents:
+      -
+        _id: 1
+        x: 11
+      -
+        _id: 2
+        x: 22
+      -
+        _id: 3
+        x: 33
+tests:
+  -
+    description: 'readConcern majority with out stage'
+    operations:
+      -
+        object: *collection_readConcern_majority
+        name: aggregate
+        arguments: &arguments
+          pipeline:
+            -
+              $sort:
+                x: 1
+            -
+              $match:
+                _id:
+                  $gt: 1
+            -
+              $out: &output_collection other_test_collection
+    expectEvents:
+      -
+        client: *client0
+        events:
+          -
+            commandStartedEvent:
+              command:
+                aggregate: *collection_name
+                pipeline: &pipeline
+                  - { $sort: { x: 1 } }
+                  - { $match: { _id: { $gt: 1 } } }
+                  - { $out: other_test_collection }
```

Comment:
Why is it that the converter script left this as-is from the original, but expanded the pipeline in the "Aggregate with $merge" test? IMO, the concise format we have here is preferred.

Reply:
The symfony/yaml library can only switch to inline dumping based on the level and no other criteria. In addition, it does not support this concise format out of the box, and this may have been one of the few places where I bothered to reformat the output.

I'd have to do some testing to make sure I'm not breaking anything, but in theory I could replace `-\n` with a single `-` and drop all but one of the spaces that follow it. Alternatively, we could try to find a tool that is able to format YAML files how we want them and enforce that across the board. I'm not aware of any such tool, but maybe someone else is.

Reply:
No worries. If this is something you can easily fix with a search/replace feel free; otherwise, I think we can just get to it as needed when we start modifying the converted files down the line.

Reply:
I found [yamlfmt](https://pypi.org/project/yamlfmt/) which can format the files in place, but I dislike the indentation it suggests:
```diff
 runOnRequirements:
-  -
-    minServerVersion: 4.1.11
+- minServerVersion: 4.1.11
```

If we're fine with that style, I'm happy to apply this to all generated files. This could also be done separately, so no need for a decision now.

Reply:
Very odd that it doesn't indent. Let's punt on this for now.